lib/ansible/config/base.yml

```diff
@@ -1746,17 +1769,7 @@ MODULE_IGNORE_EXTS:
   ini:
   - {key: module_ignore_exts, section: defaults}
   type: list
-MODULE_STRICT_UTF8_RESPONSE:
```

Comment:
@bcoca Seems you removed that by accident?

Reply:
was in the merge, thought that was removed from devel